tests/perception/render_perception_graph.py

```diff
@@ -0,0 +1,38 @@
+from adam.curriculum.curriculum_utils import PHASE1_CHOOSER
+from adam.ontology.phase1_ontology import GAILA_PHASE_1_ONTOLOGY, BALL
+from adam.perception.high_level_semantics_situation_to_developmental_primitive_perception import (
+    GAILA_PHASE_1_PERCEPTION_GENERATOR,
+)
+from adam.perception.perception_graph import PerceptionGraph
+from adam.random_utils import RandomChooser
+from adam.situation.templates.phase1_templates import (
+    Phase1SituationTemplate,
+    all_possible,
+    object_variable,
+)
+
+
+def render_perception_graph():
```

Comment:
Do we need to check this in?

Reply:
@gabbard What do you mean by that - to submit to Darpa? I can move it to a higher level directory if you prefer.

Reply:
@denizbeser : I meant rather "does this need to be kept in the repository"?  Every entry point we have adds to the cognitive burden of someone getting acquainted with the project, so if the program doesn't serve any ongoing purpose, we can remove it from the repo.

Reply:
@gabbard I see. I will keep it locally for debugging purposes and remove it from the repo then.